stuff/cron/update_ranks.py

```diff
@@ -3,10 +3,12 @@
 '''Updates the user ranking.'''
 
 import argparse
+import datetime
 import logging
 import os
 import sys
 from typing import Sequence, NamedTuple
+from dateutil.relativedelta import relativedelta
```

Comment:
nit:

```suggestion
import dateutil.relativedelta
```

y usa `datetime.relativedelta.relativedelta`, porque sólo se usa una vez.